src/components/Guilds/Sidebar/GuestActions.tsx

```diff
@@ -0,0 +1,141 @@
+import styled, { css } from 'styled-components';
+import { useState } from 'react';
+
+import { Button } from 'components/Guilds/common/Button';
+import { Modal } from 'components/Modal';
+import { Heading } from '../common/Typography';
+
+import dxIcon from '../../../assets/images/dxdao-icon.svg';
+import { FiArrowRight, FiInfo } from 'react-icons/fi';
+
+const ModalHeader = styled.div`
+  border-bottom: 1px solid #ccc;
+`;
+const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+  color: black;
+  @media only screen and (min-width: 769px) {
+    padding: 40px;
+  }
+`;
+
+const DaoBrand = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+const DaoIcon = styled.img`
+  height: 3rem;
+  width: 3rem;
+`;
+
+const DaoTitle = styled(Heading)`
+  margin-left: 4px;
+  line-height: 1;
+`;
+
+const InfoItem = styled.p`
+  margin: 0px 0px 4px 0px;
+  font-size: 14px;
+`;
+
+const BalanceWidget = styled.div`
+  width: 100%;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  margin-top: 1rem;
+  border: 1px solid ${({ theme }) => theme.colors.muted};
+  border-radius: ${({ theme }) => theme.radii.curved};
+`;
+
+const InfoRow = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 10px;
+`;
+
+const BaseFont = css`
+  font-size: 12px;
+
+  @media only screen and (min-width: 769px) {
+    font-size: 14px;
+  }
+`;
+
+const InfoLabel = styled.span`
+  ${BaseFont}
+`;
+
+const InfoValue = styled.span`
+  ${BaseFont}
+
+  font-weight: bold;
+  flex-wrap: wrap;
+`;
+const DXDValue = styled.span`
+  font-size: 20px;
+  font-weight: 600;
+`;
+
+const ButtonLock = styled(Button)`
+  width: 100%;
+  margin-top: 22px;
+`;
+
+export const GuestActions = ({ onJoin }) => {

```

Comment:
Could we make the padding more like the figma design in this component?
Currently a large space below the header and very cramped at the bottom